### PR TITLE
Postgres and cassandra setup

### PR DESCRIPTION
**Pull request by michaelrtobias**
@chanceknguyen

### CODE REVIEW

File: db/cassandra/schemaCassandra.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    releaseDate text,
+    steamAcheivments text,
+    languages text,
+    achievements  boolean,
+    partialControllersupport  boolean,
+    remotePlay boolean,
+    moreLikeThis text,
+    PRIMARY KEY (id))');
+  await db.execute('COPY games.game (id, titleCover, title, price, aboutInfo, os, processor, memory, graphics, directX, storage, genre, developer, publisher, releaseDate, steamAcheivments, languages, achievements, partialControllersupport, remotePlay, moreLikeThis) FROM '/Users/michael/Documents/Galvanize/HRR48/SDC/SDC-Project/body/gamesCassandra.csv' WITH DELIMITER='|' AND HEADER = TRUE');
+}
```
Comment (chanceknguyen) [inline db/cassandra/schemaCassandra.js:35]: Is this the doing the same thing as the schema.cql file? Is it necessary to have both?

File: db/postgres/schema.sql [hunk 2]
```diff
@@ -1,52 +1,39 @@
 DROP DATABASE IF EXISTS sdc;
 
 CREATE DATABASE sdc;
+\c sdc;
 
--- \conect sdc;
+SET maintenance_work_mem to '16GB';
```
Comment (chanceknguyen) [inline db/postgres/schema.sql:6]: Will this work on an EC2 instance?

**Review by chanceknguyen — APPROVED** [review]
Besides small comments, this LGTM

File: db/postgres/schema.sql (hunk 2 above)
Comment (michaelrtobias) [reply db/postgres/schema.sql:6]: probably not. And i dont really need that there. It doesnt speed things up that much.

File: db/cassandra/schemaCassandra.js (hunk 1 above)
Comment (michaelrtobias) [reply db/cassandra/schemaCassandra.js:35]: the schema cql file should be deleted. Forgot to clean things up

**Pull request merged**